cypress/integration/configuration/shippingMethods/shippingWeights/weightRecalculate.js

```diff
@@ -18,7 +18,7 @@ import filterTests from "../../../../support/filterTests";
 import { changeWeightUnit } from "../../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Recalculate weights", () => {
+  describe("As a staff user I want to change shop default weight unit", () => {
```

Comment:
This test fails during execution, probably changes to UI were made which resulted in error
![image](https://user-images.githubusercontent.com/97674352/153202324-2900e2db-b5bb-4b62-b6fd-12909d501abe.png)